scripts/docs.angularjs.org-firebase/readme.firebase.docs.md

```diff
@@ -13,9 +13,13 @@ See .circleci/config.yml for the complete deployment config and build steps.
 
 # Serving locally:
 
+- Run `grunt:package`.
```

Comment:
Shouldn't these commands be `yarn grunt package` (or at least `grunt package`)?
(Similarly, `yarn grunt prepareDeploy` below.)

Reply:
Yep!